content/posts/readme-html-deprecated.md

```diff
@@ -0,0 +1,40 @@
+title: Stop redirecting root documentation URL to "README.html"
+date: 2024-05-27
+category: Changelog
+tags: hosting, deprecation
+authors: Manuel Kaufmann
+status: published
+image: /images/readme-html-deprecated.jpg
+image_credit: Photo by <a href="https://unsplash.com/@introspectivedsgn?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Erik Mclean</a> on <a href="https://unsplash.com/photos/black-asus-laptop-computer-showing-3-00-sxiSod0tyYQ?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+We are announcing the **deprecation of redirecting root URLs (`/`) to `/README.html`**
+for all the projects that don't output an `index.html` file when building the documentation.
+The deprecation date is set on **Monday, July 1st, 2024**.
+After this date, projects not outputting an `index.html` file will [fail their builds](https://blog.readthedocs.com/builds-without-index/).
+
+### Deprecation timeline
+
+We understand this change will affect a few hundred of projects, so we have a timeline to communicate this deprecation to our users effectively.
+
+* **Monday, May 27, 2024**: Email project owners using this deprecated feature so they can take action.
```

Comment:
This is a US holiday, btw. Might aim for Tuesday?

Reply:
Sounds good 👍🏼